test/chain-test.js

```diff
@@ -694,79 +711,144 @@ describe('Chain', function() {
       'bad-txns-txouttotal-toolarge');
   });
 
-  it('should mine 111 multisig blocks', async () => {
-    const flags = common.flags.DEFAULT_FLAGS & ~common.flags.VERIFY_POW;
+  describe('Sigops', function() {
```

Comment:
This test was somewhat hard to follow because of many numbers, maybe general comment would help.

To summarize: we have 80 blocks with 100 outputs and each outputs has 100 checksigs(sigop),
 but to construct test blocks (max and max + 1) we use 80 blocks but only 10 transactions from each so we get (80 * 10 * 100 = 80000 sigops)

Maybe just summary of the process and mention of 80k sigops being max in the beginning would help - or choosing simpler to follow numbers. I was confused when I got 800k sigops from in the generation block (80 block * 100 outputs * 100 sigops = 800k).

For simpler numbers, not sure. Maybe just generate 10 txs or reduce sigops to 10 instead. Maybe leave as it is just do above, add description.